Assets/Tests/InputSystem/CoreTests_Actions.cs

```diff
@@ -9550,6 +9549,36 @@ public void Actions_CanResolveActionReference()
 
         Assert.That(referencedAction, Is.SameAs(action2));
     }
+    
+    [Test]
+    [Category("Actions")]
+    public void Actions_CanResolveActionReference_WhenUsingToInputActionToConstructANewReference()
+    {
+        var map = new InputActionMap("map");
+        map.AddAction("action1");
+        var action2 = map.AddAction("action2");
+        var asset = ScriptableObject.CreateInstance<InputActionAsset>();
+        asset.AddActionMap(map);
+        
+        var reference = ScriptableObject.CreateInstance<InputActionReference>();
+        reference.Set(asset, "map", "action2");
+        
+        var copy1 = InputActionReference.Create(reference.action);
+        var copy2 = InputActionReference.Create(reference.ToInputAction());
+        
+        // Expecting action to be the same
+        Assert.That(reference.action, Is.SameAs(copy1.action));
+        Assert.That(reference.action, Is.SameAs(copy2.action));
+    }
+
+    [Test]
+    [Category("Actions")]
+    public void Actions_CanImplicitlyConvertReferenceToAction_WhenAssigningActionFromReference()
+    {
+        var reference = ScriptableObject.CreateInstance<InputActionReference>();
+        InputAction action = reference; // implicit conversion
+        Assert.That(reference.action, Is.Null);
+    }
```

Comment:
It's unclear to me why we would need these tests in this PR as it is unrelated to the object picker. Maybe I'm missing something or this is just a test.